pkg/machines/hostvmslist.jsx

```diff
@@ -465,8 +441,30 @@ class HostVmsList extends React.Component {
             allActions = allActions.concat(actions);
         }
 
+        const errors = [...ui.errors];
+        const isLibvirtActive = libvirtState.status === 'active';
+
+        if (!isLibvirtActive) {
+            errors.push({
+                message: "Libvirt is not active.",
+                actionMessage: "Start libvirt to fix this problem",
+                actionCockpitLink: "/system/services#/libvirtd.service",
+            });
+        }
+
+        let notificationArea = errors.length > 0 ? (
```

Comment:
Can we move this into the `NotificationArea` component, so it can properly handle even empty list?